packages/toasts/src/Toasts.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from 'react';
+import { Toast } from '@edx/paragon';
+
+import { ToastsContext } from './ToastsProvider';
+
+const Toasts = () => {
+  const { toasts, removeToast } = useContext(ToastsContext);
+  return toasts.map(({ id, message }) => (
+    <Toast
+      key={id}
+      onClose={() => removeToast(id)}
+      show
```

Comment:
So, yeah... haha. This is exactly how it is currently implemented in original implementation but after a closer look, I realized I think we actually may be using `Toast` from Paragon incorrectly...

As is, we are rendering `Toast` immediately when we want it to be displayed via `addToast` in the MFE code, by forcing `show={true}` right away. The issue with this is that we actually lose the CSS animation of toasts fading in/out with this approach, unfortunately 😢 

Instead, I believe we may actually want to render the `Toast` component even before its officially "visible", so that the parent container element Paragon has a chance to render first before any `Toasts` are visible. That way, when a `Toast` flips from `show={false}` to `show={true}`, it can animate properly.

By waiting to render a `Toast` only when we want it to be visible (where `show` always is true) means the toast container is not in the DOM initially. By the time a `Toast` is rendered when `addToast` is called, it's both adding the toast container _and_ the individual toast itself simultaneously, which doesn't give the browser an opportunity to do its CSS animations for the now visible toast.

Check out the fade in/out animation effect on the [Paragon documentation site](https://paragon-openedx.netlify.app/components/toast/#basic-usage). You'll notice in our enterprise learner/admin portal MFEs, wherever we call `addToast` in its current implementation, there is no animation.


It's definitely an open question still in terms of how (or whether) we might want to address this, though... 

@brobro10000 Sorry, throwing a lot of info at you about how the original implementation we're basing this PR on wasn't solid to begin with... would love to get your take on this 😉